librz/core/cmd_type.c

```diff
@@ -1010,15 +1076,15 @@ RZ_IPI int rz_cmd_type(void *data, const char *input) {
 			if (name && type) {
 				name++; // skip the '.'
 				if (rz_str_startswith(type, "struct")) {
-					rz_core_cmdf(core, "tsc %s", name);
+					print_struct_union_in_c_format(TDB, stdifstruct, rz_str_trim_head_ro(input + 2), true);
```

Comment:
shouldn't you just use `name`?
```suggestion
					print_struct_union_in_c_format(TDB, stdifstruct, name, true);
```

Reply:
Ah, sorry, yes.
Also, shouldn't that freed after this `if` block, too?